webpack/assets/javascripts/react_app/redux/API/APIRequest.js

```diff
@@ -21,6 +33,11 @@ export const get = async (
       response: data,
     });
   } catch (error) {
+    const doesIntervalExist =
+      interval !== undefined && selectDoesIntervalExist(getState(), key);
```

Comment:
nit: the undefind comparing can be removed
```js
const doesIntervalExist interval && selectDoesIntervalExist(getState(), key);
```
can we just use the selector? if the key doesn't exist it'll return false anyway, right?

Reply:
thanks I'll remove the comparing, though I prefer still to check if the `action.interval` exist,
to understand if this action is using the interval middleware.. if it's not, there is a small chance that there is another action which is using the interval with the same key as the API key, and we could accidently delete it, so I prefer to be more certain.